recipes/googletrans/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "googletrans" %}
+{% set version = "2.3.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "49cce281c5ded86fc1ef0429e240da650bae2dd7939dc5e664d80c33fa8ba3b0" %}
+
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch_python: True
```

Comment:
Use `noarch: python` instead of this. You can do this in the feedstock and rerender.